columnar/Cargo.toml

```diff
@@ -33,6 +33,6 @@ harness = false
 name = "bench_access"
 harness = false
 
-
 [features]
 unstable = []
+zstd-compression = ["sstable/zstd-compression"]
```

Comment:
that should be a default feature flag

Reply:
Are you saying I should make it a default flag and then eliminate the changelog entry so we can merge this PR?

Reply:
yes

Reply:
understood, pushed